### PR TITLE
FIX - Floating Beds

### PR DESCRIPTION
**Pull request by ghost**
This will fix the floating beds issue ya'll have been having in multiplayer mode.

### CODE REVIEW

**Comment by glitchdetector** [comment]
Can't exactly be picky with contributions since they're already a great effort by someone else.
There's a few things in there I'd question, but hey, thanks a ton!

File: flatbed.lua [hunk 1]
```diff
@@ -44,6 +44,130 @@ AddTextEntry(controllerMessageLoweredCar, "Press ~INPUT_CONTEXT~ to ~y~raise ~w~
...
+	for obj in EnumerateObjects() do
+		table.insert(objs, obj)
+	end
+
+	return objs
+end
+
+function BigDelete(entity) 
+    local playerPed = PlayerPedId()
+    carModel = GetEntityModel(entity)
```
Comment (glitchdetector) [inline flatbed.lua:118]: Globally scoped values for no apparent reason

File: flatbed.lua [hunk 2]
```diff
@@ -44,6 +44,130 @@ AddTextEntry(controllerMessageLoweredCar, "Press ~INPUT_CONTEXT~ to ~y~raise ~w~
...
+
+        SetEntityAsMissionEntity(entity, true, true)
+        
+        local timeout = 2000
+        while timeout > 0 and not IsEntityAMissionEntity(entity) do
+            Wait(100)
+            timeout = timeout - 100
+        end
+
+        Citizen.InvokeNative( 0xEA386986E786A54F, Citizen.PointerValueIntInitialized( entity ) )
```
Comment (glitchdetector) [inline flatbed.lua:137]: Invoking `DeleteVehicle` as if we're in 2016

**Comment by ghost** [comment]
Oopsies, I found that enumerator boilerplate somewhere; forgot to remove and localize some functions

**Comment by ghost** [comment]
If you want ill push a fix for it

**Pull request merged**